packages/formStructure/readme/fieldList.md

```diff
@@ -0,0 +1,5 @@
+# FieldList
+
+A `FieldList` is used to input data for enumerable objects. For example: defining key-value pairs.
```

Comment:
another example you can give is inviting users and capturing first name, last name, and email address (we are doing this in conductor and can get screenshot from Nan.
(although we have group too as a field ;)

Let's raise the guidance to 4.

Reply:
> Let's raise the guidance to 4.

What do you mean by this?

Reply:
@mperrotti  current text says 3 so suggesting 4. 
"Avoid putting too many fields in a field group, ideally no more than 3."